utils/bignum/int.go

```diff
@@ -0,0 +1,62 @@
+package bignum
+
+import (
+	"crypto/rand"
+	"fmt"
+	"io"
+	"math/big"
+)
+
+func NewInt(x interface{}) (y *big.Int) {
```

Comment:
Would be good to see the accepted types in the godoc

Reply:
Done